tests/scripts/all.sh

```diff
@@ -4391,6 +4391,63 @@ component_build_aes_aesce_armcc () {
     armc6_build_test "-O1 --target=aarch64-arm-none-eabi -march=armv8-a+crypto"
 }
 
+support_build_aes_armce() {
+    # clang >= 4 is required to build with AES extensions
+    ver="$(clang --version|grep version|sed -E 's#.*version ([0-9]+).*#\1#')"
+    [ "${ver}" -ge 11 ]
```

Comment:
Bah, missed that this doesn't match the comment two lines above. I have a PR touching this function that I'll fix this in